Makefile

```diff
@@ -40,7 +40,7 @@ docs:  ## Build the docs using Sphinx
 	cd docs && make clean html && cd .. && echo "Documentation built in $(shell pwd)/docs/_build/html/index.html"
 
 restart: ## Restart Triggerer & Scheduler container
-	docker-compose -f dev/docker-compose.yaml restart airflow-triggerer airflow-scheduler
+	docker-compose -f dev/docker-compose.yaml restart airflow-triggerer airflow-scheduler airflow-worker
```

Comment:
you could have user restart-all, instead of adding worker in this

Reply:
`restart-all` restarts all the services including postgres, webserver, redis & flower which take quite some time to restart. However, restarting the `worker` is quick and is a need to quickly restart the worker to check debug logs added during development.